docs/conf.py

```diff
@@ -76,6 +76,7 @@
 }
 html_static_path = ["_static"]
 html_css_files = []
+html_context = {"analytics_enabled": True}
```

Comment:
We should only enable analytics in docs jobs when we're publishing to qiskit.org we don't want to include any of this by default. It should be disabled here and for all builds except for those run as part of a publish job.

Reply:
What would you suggest? I'm not familiar with the terra build process, but this is the important info about analytics:
- no analytics are collected unless the page is loaded at a qiskit.org domain, so even running locally etc. will not trigger any analytics
- the point of the flag here is to give qiskit projects control over whether they collect analytics data once deployed to qiskit.org. So if this line is removed even if an app is using the latest theme version they will not be collecting analytics data by default even once deployed to qiskit.org

Reply:
This conf.py isn't used at all for published documentation so we should not set it at all in the terra repo. It should only be set in qiskit/qiskit, I also would only set it via an env variable that gets set during the deployment jobs.

Reply:
ah gotcha, I didn't realise the `conf.py` here wasn't even used